thoth/prescriptions_refresh/handlers/quay/security.py

```diff
@@ -165,11 +165,21 @@ def _create_vulnerability_prescriptions(image: str, tag: str, vulnerabilities: L
 
         cve_seen.add(vulnerability["Name"])
 
+        if "Link" in vulnerability.keys():
+            vulnerability_link = vulnerability["Link"]
+            message = f"Found vulnerability {vulnerability['Name']!r} in the base image used."
+            " See the link provided for more information."
+
+        else:
+            vulnerability_link = "https://thoth-station.ninja/j/no_cve_link_provided"
```

Comment:
You can directly use the following link that internally translates to our justifications:

```suggestion
            vulnerability_link = "no_cve_link_provided"
```